File: debop4s-core/src/main/scala/kr/debop4s/core/SortDirection.scala
package kr.debop4s.core

/**
 * 정렬 방법
 *
 * @author 배성혁 sunghyouk.bae@gmail.com
 * @since  2013. 12. 9. 오후 10:25
 */
object SortDirection extends Enumeration {

    type SortDirection = Value

    val ASC = Value(1, "ASC")

    val DESC = Value(-1, "DESC")
}

File: debop4s-core/src/main/scala/debop4s/core/SortDirection.java
package debop4s.core;

/**
 * 정렬 방법
 *
 * @author 배성혁 ( sunghyouk.bae@gmail.com )
 * @since 12. 12. 19.
 */
public enum SortDirection {

    /** 순차 정렬 */
    ASC("ASC"),

    /** 역순 정렬 */
    DESC("DESC");

    private final String value;

    public String getValue() {
        return value;
    }

    SortDirection(String value) {
        this.value = value;
    }

}
